helm-chart/eoapi/values.yaml

```diff
@@ -334,5 +334,18 @@ vector:
       # https://www.uvicorn.org/settings/#production
       WEB_CONCURRENCY: "5"
 
+######################
+# STAC Browser
+######################
+# It is a good idea to deploy stac-browser outside of k8s, since it's SPA with static files.
+# Please consider alternatives, such as cloud storage + CDN, for example 
+browser:
+  enabled: true
+  replicaCount: 1
+  image:
+    # we use a custom image that overrides pathPrefix value
+    name: ghcr.io/alekzvik/stac-browser-prefix
```

Comment:
I don't think it is good to pull a custom made image from a personal source.

Reply:
Now we build our own stac browser image in the CI/CD to have the pathPrefix set to /browser/